PyInstaller/hooks/hook-pyopencl.py

```diff
@@ -0,0 +1,13 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2015-2016, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+# Hook for the pyopencl module: https://github.com/pyopencl/pyopencl
+
+from PyInstaller.utils.hooks import copy_metadata
+datas = copy_metadata('pyopencl')
```

Comment:
Yes, I closed #2305. But that one doffers from yout PR, this is why I'm asking you about collecting the data-files which is done there.